src/App.jsx

```diff
@@ -14,70 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodValue, setGoodValue] = useState('Jam');
```

Comment:
goodValue is a bad name, because there is a select good in this variable, so I suggest you to rename it to selectedGood
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```